tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.csproj

```diff
@@ -19,9 +23,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="public_privatekey.pfx" />
```

Comment:
The name of the certificate is a little generic. Could it be renamed to something to provide a better context? e.g IdentitySigningKey?